api/src/certification/enrolment/domain/models/Candidate.js

```diff
@@ -47,9 +48,14 @@ export class Candidate {
     this.complementaryCertificationId = complementaryCertificationId;
     this.billingMode = billingMode;
     this.prepaymentCode = prepaymentCode;
+    this.hasSeenCertificationInstructions = hasSeenCertificationInstructions;
   }
 
   isLinkedToAUser() {
     return !_.isNil(this.userId);
   }
+
+  validateCertificationInstructions() {
+    this.hasSeenCertificationInstructions = true;
```

Comment:
Pour renforcer un peu les règles métier autour de la manipulation de ce switch

```suggestion
  if(authorizedToStart) {
    this.hasSeenCertificationInstructions = true;
  }
```

Reply:
Il n'est pas forcément autorisé à commencer à ce niveau là, c'est avant le code candidat